docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -63,7 +63,7 @@ trait IERC721 {
 :eip165: https://eips.ethereum.org/EIPS/eip-165[EIP-165]
 
 `ISRC5`, from the {snip5} standard, allows for other contracts to query if the implementing contract supports a specific interface.
-This is especially important with NFT contracts because if an NFT is transferred to a contract that does not support the `IERC721` interface, the NFT is essentially lost forever.
+This is especially important with NFT contracts because if an NFT is transferred to a contract that does not support the `IERC721` interface, the NFT may be lost forever.
```

Comment:
```suggestion
This is especially important with NFT contracts because if an NFT is transferred to a contract that does not support the `IERC721Receiver` interface, the NFT may be lost forever.
```